server/datastore/mysql/migrations/tables/20220107155700_RemoveHostIDFK.go

```diff
@@ -0,0 +1,47 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220107155700, Down_20220107155700)
+}
+
+func Up_20220107155700(tx *sql.Tx) error {
+	tables := []string{
+		"host_additional",
+		"host_users",
+		"policy_membership",
+	}
+	for _, table := range tables {
+		if err := removeHostIDFK(tx, table); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func removeHostIDFK(tx *sql.Tx, table string) error {
+	referencedTables := map[string]struct{}{
+		"hosts": {},
+	}
+	constraints, err := constraintsForTable(tx, table, referencedTables)
+	if err != nil {
+		return errors.Wrap(err, "getting references to hosts table")
+	}
```

Comment:
Nit: You could check the expected `len(constraints) == 3`, to be extra sure.

Reply:
Though it may cause trouble if for some reason it fails on some and the user re-runs...